### PR TITLE
UG: case insensitive tags + images

### LINKED ISSUE #206: UG: Update case-insensitivity of tags
* Explain that tags within a single module are case-insensitive

### PR DESCRIPTION
**Pull request by Silvernitro**
* Add in note about case-insensitive tags in `add`
* General edits after proofreading the UG
* Update screenshots for `gem`,`science`, and `search` (we now show semesters)

https://silvernitro.github.io/tp/UserGuide.html

Closes #206

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -337,7 +349,7 @@ from NUSMods database directly and displayed in the result display box.
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Note:**<br>
-If your internet is down, you can only search for modules that are of the Computer Science curriculum.
+If your internet is down, you can only search for modules that are in the NUS Computer Science curriculum.
```
Comment (yan-soon) [inline docs/UserGuide.md]: minor minor nit: can change to "not connected to internet" as you did earlier HAHAH

File: docs/UserGuide.md [hunk 2]
```diff
@@ -120,6 +123,9 @@ Detailed explanations of each commands and their respective usage are listed und
 For example, if the command specifies `edit MODULE_CODE c/NEW_MODULE_CODE t/core`, `edit MODULE_CODE t/core c/NEW_MODULE_CODE
 ` is also acceptable.
 
+* When a command does not require parameters, e.g. `help`, adding extraneous words, e.g `help me` will not affect the
+ command.
```
Comment (yan-soon) [inline docs/UserGuide.md]: MINOR MINOR nit: should we add an extra, "allowing the command to run as per normal" elaboration to make it clearer?

**Review by yan-soon — APPROVED** [review]
LGTM!!!!!!!!!!!!!!!!!

**Review by shaokiat — APPROVED** [review]
LGTMLGTMLGTMLGTMLGTMLGTM

**Pull request merged**